src/apps/frontend/pages/authentication/forgot-password/back-button.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+interface BackButtonProps {
+  onClick: () => void;
+}
+
+const BackButton: React.FC<BackButtonProps> = ({
+  onClick,
+}) => (
+    <button
+      onClick={onClick}
+      className="mb-5 cursor-pointer text-lg transition active:text-primary/80"
```

Comment:
`mb-5`: Its not the job of the component to add margins around it, but rather the container to layout the components and put spacing between them.